bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/internal/findandreplace/overlay/FindReplaceOverlay.java

```diff
@@ -400,9 +422,9 @@ private void applyOverlayColors(Color color, boolean tryToColorReplaceBar) {
 		wholeWordSearchButton.setBackground(color);
 		regexSearchButton.setBackground(color);
 		caseSensitiveSearchButton.setBackground(color);
-		searchAllButton.setBackground(color);
-		searchUpButton.setBackground(color);
-		searchDownButton.setBackground(color);
+		selectAllButton.setBackground(color);
```

Comment:
Side-note: I have always been confused about whether we "search" or we "find" or we "select". I tried being consistent but I feel like there are more places where we have inconsistencies.

Reply:
In particular for "search" and "find" there is probably no sharp distinction. I propose to simply resolve imprecise names whenever we find them.